releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,17 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.2.0 - 
+
+### New Features
+* Added support for feature flags, which allows feature flag settings to be saved to a Kubernetes ConfigMap.
+    * Added the `featureFlag.selectors` property for feature flag filtering.

```

Comment:
Added support for feature flags, which allows projecting feature flags to the target ConfigMap.
  * Added the `spec.featureFlag.selectors` property for filtering the feature flags by multiple combinations of `keyFilter` and `labelFilter` .
  * Added the `sepc.featureFlag.refresh` property for periodically updating feature flags with the latest data from App Configuration.

Reply:
This is a bit too many details. We can point people to the documentation. The release notes aren't documentation.